be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -431,7 +431,6 @@ Status ExchangeSinkOperator::prepare(RuntimeState* state) {
     // Randomize the order we open/transmit to channels to avoid thundering herd problems.
     _channel_indices.resize(_channels.size());
     std::iota(_channel_indices.begin(), _channel_indices.end(), 0);
-    srand(reinterpret_cast<uint64_t>(this));
     std::shuffle(_channel_indices.begin(), _channel_indices.end(), std::mt19937(std::random_device()()));
```

Comment:
can add a comment here to avoid someone adding srand back again.

Reply:
Fixed it. Thanks.